### PR TITLE
ADD social media links to README

### PR DESCRIPTION
**Pull request by tomByrer**
I had to use Google to find your blog.  Maybe you're getting a cut of Google's ads?  I hope I'm not making you go broke....
;)

I'm assuming the Discord channel is still internal-only?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,6 +151,13 @@ We have a badge ! Link it to your live Storybook example.
 [![Storybook](https://github.com/storybooks/press/blob/master/badges/storybook.svg)](link to site)
 ```
 
+## Community
+
+-   Tweeting via [@storybookjs](https://twitter.com/storybookjs)
+-   Blogging at [Medium](https://medium.com/storybookjs)
+-   Chatting on [Slack](https://now-examples-slackin-rrirkqohko.now.sh/)
```
Comment (pksunkara) [inline README.md]: Let's use Discord here.

Comment (tomByrer) [reply README.md]: Sure, have the URL please?

Comment (pksunkara) [reply README.md]: https://discord.gg/sMFvFsG

Comment (pksunkara) [reply README.md]: My bad. I meant, add discord here. Let's have both slack and discord.

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Thank you @tomByrer !

**Comment by tomByrer** [comment]
Welcome

**Pull request merged**